k8s/operators/src/lib.rs

```diff
@@ -1,2 +1,59 @@
-/// The DiskPool operator
+use crate::error::Error;
+use kube::Client;
+use tracing::{info, warn};
+
+/// The DiskPool operator.
 pub mod diskpool;
+/// The error module for the operator.
+pub mod error;
+/// The MayaStorPool Operator definitions.
+pub mod mayastorpool;
+
+/// Migrate from MayastorPool
+pub async fn migrate_and_clean_msps(k8s: &Client, namespace: &str) -> Result<(), Error> {
```

Comment:
Shouldn't this be part of the binary rather than the library?

Reply:
So the library is only here for the support tool to inspect the DiskPool CR IIRC. We can move the rest to diskpool which is only used by the diskpool binary IIRC

Reply:
Done. Moved this to binary. And the only binary module exposed via lib is crd.